apiml-common/src/main/java/com/ca/mfaas/product/service/ServiceStartupEventHandler.java

```diff
@@ -20,19 +20,21 @@
 
 public class ServiceStartupEventHandler {
     public static final int DEFAULT_DELAY_FACTOR = 5;
-    private final ApimlLogger apimlLog = ApimlLogger.of(ServiceStartupEventHandler.class, YamlMessageServiceInstance.getInstance());
+    private final ApimlLogger apimlLog = ApimlLogger.of(ServiceStartupEventHandler.class,
+            YamlMessageServiceInstance.getInstance());
 
     @SuppressWarnings("squid:S1172")
     public void onServiceStartup(String serviceName, int delayFactor) {
```

Comment:
`int delayFactor` is never used even tho is passed

Reply:
Hello Andrea @taban03, thank you for noticing it. It is in the code that we have not to change but only formatted. But it is good to notice these things and we will improve the code as the boy-scout rule says: _always leave the code behind in a better state than you found it_. (https://martinfowler.com/bliki/OpportunisticRefactoring.html)

.

Reply:
The delayFactor is now used. Thank you, Andrea, for noticing it!